src/octokit.ts

```diff
@@ -16,18 +23,33 @@ type Event = {
 
 export const createOctokitInstance = (token: string) => TE.fromEither(E.tryCatch(() => new GitHub(token), E.toError));
 
+const tryCreateGithubCheck = async (
```

Comment:
@XVincentX Could you please help me rewrite this in a more fp-ts way? I haven't found the proper way to do this.

Reply:
You really can't, since the underlying API has side-effects. You can wrap it (as you're doing) with `tryCatch` later